firmware/protect_payload/main.rs

```diff
@@ -0,0 +1,69 @@
+//! Protect payload test
+//!
+//! This test firmware checks that Miralis is properly protected, or in other words that the
+//! firmware is properly sandboxed.
```

Comment:
A small nit: here it we don't check that Miralis is protected, we check that the protect payload policy works as expected. We should probably at least mention that the test is intended to be run with the Protect Payload policy (which is not enabled by default, so might be surprising to someone who tries to run the test with `just run protect_payload`)

Reply:
Looks like you rolled back on this file, it's the old text and old code